spec.html

```diff
@@ -37285,9 +37403,9 @@ <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
         1. Let _jsonString_ be ? ToString(_text_).
         1. [id="step-json-parse-validate"] Parse ! StringToCodePoints(_jsonString_) as a JSON text as specified in ECMA-404. Throw a *SyntaxError* exception if it is not a valid JSON text as defined in that specification.
         1. Let _scriptString_ be the string-concatenation of *"("*, _jsonString_, and *");"*.
-        1. Let _script_ be ParseText(! StringToCodePoints(_scriptString_), |Script|).
+        1. [id="step-json-parse-parse"] Let _script_ be ParseText(! StringToCodePoints(_scriptString_), |Script|).
         1. Assert: _script_ is a Parse Node.
-        1. Let _completion_ be the result of evaluating _script_. The extended PropertyDefinitionEvaluation semantics defined in <emu-xref href="#sec-__proto__-property-names-in-object-initializers"></emu-xref> must not be used during the evaluation.
+        1. Let _completion_ be the result of evaluating _script_. The extended PropertyDefinitionEvaluation semantics defined in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref> must not be used during the evaluation.
```

Comment:
```suggestion
        1. [id="step-json-parse-eval"] Let _completion_ be the result of evaluating _script_. The extended PropertyDefinitionEvaluation semantics defined in <emu-xref href="#sec-runtime-semantics-propertydefinitionevaluation"></emu-xref> must not be used during the evaluation.
```

Reply:
can you provide a concrete suggestion? my contribution here is the editorial commit; this is the normative stuff :-)

Reply:
```suggestion
        1. [id="step-json-parse-eval"] Let _completion_ be the result of evaluating _script_.
        1. NOTE: The PropertyDefinitionEvaluation semantics defined in <emu-xref href="#sec-runtime-semantics-propertydefinitionevaluation"></emu-xref> have special handling for the above evaluation.
```